### ISSUE TITLE
Nested repeating group is not visible when `largeGroup=false`

### ISSUE THREAD

**Issue by bjosttveit** [Altinn/app-frontend-react#834]
### Description of the bug

Currently, when using `largeGroup: false` (default) on a summary of a repeating group, any nested repeating groups will be hidden. This is because there is no string representation of a repeating groups data. Previously it would actually show the field with an empty value but this was removed due to confusion.

When using `largeGroup: true` the nested group will be displayed as a "compact" group.

Not sure how these kinds of summaries should be handled. Should nested groups be hidden from summary? Show just the number of elements? Is there some possible string representation of a repeating group? Or, should it show all of the content in a compact way?

A problem currently is that PDF generation uses `largeGroup: false` by default when using automatic PDF generation (not using a custom layout), and there is currently no way of overriding summary props using this method. This means that if you have a nested repeating group that you want to be visible in the PDF you are forced to make an entire custom layout. I would like to avoid making `largeGroup: true` default, as it takes up a lot more space and is not generally what is needed for most repeating groups.

This needs design work before it can be implemented.

### Steps To Reproduce

1. Create a repeating group containing a nested repeating group.
2. Create a summary for the first group, using `largeGroup: false`.
3. Fill inn information in both the first and nested repeating group.
4. Observe that the data for the nested repeating group is not present.

**Comment by bjosttveit** [comment]
Relevant slack discussion related to automatic PDF generation: https://altinn.slack.com/archives/C02EVE4RU82/p1679396382964929

**Comment by RonnyB71** [comment]
Closing, will be solved by https://github.com/Altinn/app-frontend-react/issues/2052
